src/platform/site-wide/sass/brand-consolidation.scss

```diff
@@ -34,6 +34,7 @@ $image-path: "~uswds/src/img";
 
 @import "~@department-of-veterans-affairs/formation/sass/shame";
 @import "merger";
+@import "modules/m-homepage";
```

Comment:
Started trying to pull some styles out into another file. I have another homepage ticket where I'm going to finish moving things, I think.